compensated-rails/.gitignore

```diff
@@ -7,3 +7,4 @@ spec/dummy/db/*.sqlite3-*
 spec/dummy/log/*.log
 spec/dummy/storage/
 spec/dummy/tmp/
+Gemfile.lock
```

Comment:
Including the `Gemfile.lock` is a good idea in applications, but less so in gems where we want to test against multiple versions of Ruby/Gems.